inc/parser/xhtml.php

```diff
@@ -1627,14 +1674,25 @@ public function _media($src, $title = null, $align = null, $width = null,
 
         list($ext, $mime) = mimetype($src);
         if(substr($mime, 0, 5) == 'image') {
+            $jpeg = false;
+
+            if(($ext == 'jpg' || $ext == 'jpeg') &&
+               (is_null($width) || is_null($height))) {
+                $jpeg = $this->_load_jpeg_meta($src);
+                list($width, $height) = $this->_image_sizes($width, $height, $jpeg);
+            }
+
```

Comment:
My understanding of the code is that these lines can be placed in the ` } elseif($ext == 'jpg' || $ext == 'jpeg') {` below, it cannot?

Reply:
Originally I tried that, but it turns out the code cannot be placed there: the conditional below only runs when the title is unset and will be loaded from the image file.

The code I’m introducing should be run independently of where the title comes from.